.github/workflows/dev_pr/labeler.yml

```diff
@@ -47,4 +47,4 @@ substrait:
   - datafusion/substrait/**/*
 
 sqllogictest:
-  - datafusion/core/tests/sqllogictests/**/*
```

Comment:
This illustrates that the sqllogictest code is no longer so deeply nested in the tree, which I think is a good change overall